### PR TITLE
feat: support latest version serialport

### PR DESCRIPTION
**Pull request by chaos2171053**
@kolonist  support serilaport latest version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,17 +50,19 @@ All asynchronous functions in this library return Promises.
 ## `Port`
 Class serves connection to radio-module, sends AT-commands and receives answers.
 
-## `constructor (name [, options])`
+## `constructor ([, options])`
```
Comment (kolonist) [inline README.md]: `([options])` without `,`

Comment (kolonist) [reply README.md]: probably without square braces

**Review by kolonist — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (liangchaoqin) [reply README.md:53]: 👌,i will fix it

Comment (chaos2171053) [reply README.md]: ok,i will fix it

**Review by liangchaoqin — APPROVED** [review]

**Comment by kolonist** [comment]
Thank you!

I will add it to npm a bit later

**Comment by kolonist** [comment]
npm package updated

**Pull request merged**